generator/fluentReportsGenerator.js

```diff
@@ -3013,7 +3049,7 @@ class frElement { // jshint ignore:line
      * Delete this Element
      */
     delete() {
-        if (this._report.currentSelected === this) {
+        if (this._report.currentSelected[0] === this) {
```

Comment:
This should be updated to be something like:
`if (this._report.currentSelected.indexOf(this) >= 0)`  As you want to know if it is ANY of the selected elements.

Reply:
Actually no, see the `Delete` & `Duplicate` buttons were removed for Multible Elements, (the hotkeys weren't however)
So `delete` and `duplicate` can _only_ run if there is only one elem in `currentSelected`

Reply:
I only deleted the `Delete` & `Duplicate` buttons just because it felt like they shouldn't be there. I got no reason why they can't be re-added.

Reply:
We can leave the buttons off, but  I think the code should be updated to work if it is anywhere in the currentSelected list their is no reason to leave the code broken even if not used.